src/main/java/io/anserini/document/TwitterDocument.java

```diff
@@ -0,0 +1,63 @@
+package io.anserini.document;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+
+/**
+ * A Twitter document (status).
+ */
+public class TwitterDocument implements SourceDocument {
+  private static final Logger LOG = LogManager.getLogger(io.anserini.document.twitter.Status.class);
+
+  private String id;
+  private String contents;
+
+  private static final JsonParser JSON_PARSER = new JsonParser();
```

Comment:
This is just a personal preference, so don't feel obligated to change, but I'd put the `static` set first.